### DISCUSSION TITLE
Publico objetivo

### DISCUSSION THREAD

**Discussion by PaulinaTorrecillas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#79] [Q&A] [answered]
Hola, tengo dudas de como haremos lo del publico objetivo en la plantilla que el comander nos dejo, si alguien puede ayudarme se lo agradecería mucho

**Comment by doguedogue** [comment, marked as answer]
La página que dió de referencia explica mucho, vas poniendo a quienes les vas a vender tu producto o servicio.

Ponte en el papel que eres la que vende el sitio web, quienes usan a abogados?adónde están? Etc. Plantéate esas preguntas, tu puedes.

**Reply by PaulinaTorrecillas** [reply]
Okey ya entendí gracias!!
